### ISSUE TITLE
PowerShell Startup Exception after updating to 7.2.0-preview.9

### ISSUE THREAD

**Issue by rmnp30** [PowerShell/PSReadLine#2793]
### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0-preview.9
OS: Microsoft Windows 10.0.19044
BufferWidth: 130
BufferHeight: 9001

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004ca+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x600049b+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000101+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000fa+0x60
```

**Comment by rmnp30** [comment]
I was able to resolve this exception by re-installing 7.2.0-preview.5 and then manually installing PSReadLine 2.2.0-beta3. I initially ran into the following error during the PSReadLine update, even though I was running Powershell as an Administrator: "Install-Package: Administrator rights are required to install or update." To resolve this I renamed the PSReadLine module within $env:homepath\Documents\Powershell\Modules and then re-executed: Install-Module -Name PSReadLine -Force -AllowPrerelease. I verified PSReadLine 2.2.0-beta 3 was installed using: Get-InstalledModule PSReadLine. After this I re-ran the MSI associated with 7.2.0-preview.9 and tested Powershell 7 without issue.
